llvm/include/llvm/Frontend/OpenMP/OMP.td

```diff
@@ -510,6 +528,18 @@ def OMP_EndAssumes : Directive<"end assumes"> {
   let association = AS_Delimited;
   let category = OMP_Assumes.category;
 }
+def OMP_Assume : Directive<"assume"> {
+  let association = AS_Block;
```

Comment:
why need this?

Reply:
I don't understand the question. Do you think the .td fragment is unnecessary for adding support for the "omp assume" directive? There is no comment here, but that follows the general trend in that file.